lib/matplotlib/tests/test_axes.py

```diff
@@ -6420,3 +6420,16 @@ def test_polar_interpolation_steps_variable_r(fig_test, fig_ref):
     l.get_path()._interpolation_steps = 100
     fig_ref.add_subplot(projection="polar").plot(
         np.linspace(0, np.pi/2, 101), np.linspace(1, 2, 101))
+
+def test_normalize_kwarg_warn_pie():
+    fig,ax = plt.subplots()
+    with pytest.warns(MatplotlibDeprecationWarning):
+        ax.pie(x=[0], normalize=None)
+
+def test_normalize_kwarg_pie():
+    fig,ax = plt.subplots()
```

Comment:
```suggestion
    fig, ax = plt.subplots()
```
Please watch the style. flake8 will complain about this.